pages/scheduleManager.tsx

```diff
@@ -188,14 +187,21 @@ const TaskName = styled.div`
 `;
 
 const EditButton = styled.button`
+<<<<<<< HEAD
+  width: 5vw;
+=======
   width: 30px + 1vw;
+>>>>>>> staging
   float: right;
   color: red;
   background-color: white;
   align-self: flex-start;
   border-radius: 5px;
   border: 2px solid red;
+<<<<<<< HEAD
```

Comment:
merge conflict fyi @abigger87

Reply:
u mind making a hotfix? can't get to it rn